exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/okhttp/JsonRequestBody.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.internal.okhttp;
+
+import io.opentelemetry.exporter.internal.marshal.Marshaler;
+import io.opentelemetry.exporter.internal.marshal.MarshalerUtil;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import okhttp3.MediaType;
+import okhttp3.RequestBody;
+import okio.BufferedSink;
+
+public class JsonRequestBody extends RequestBody {
```

Comment:
Pretty much a copy paste of ProtoResponseBody

Reply:
I believe this can be package private, and final. (`ProtoResponseBody` can be package private as well.)